recipes/r-optmatch/meta.yaml

```
@@ -0,0 +1,103 @@
+{% set version = '0.10.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-optmatch
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/optmatch_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/optmatch/optmatch_{{ version }}.tar.gz
+  sha256: 2b7661567fdfbdb39dae3779b4b7dea2b9eb759d3117311b3a7936b10cce1d35
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'  # [win]
+    - '*/Rblas.dll'  # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp
+    - r-dplyr
+    - r-rlemon
+    - r-tibble
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp
+    - r-dplyr
+    - r-rlemon
+    - r-tibble
+    - libopenblas
```

```suggestion
    - libblas
```
Let's see if this works first.